docs/en/stack/ml/df-analytics/flightdata-classification.asciidoc

```diff
@@ -0,0 +1,484 @@
+--------------------------------------------------
+// TEST[skip:TBD]
+
+
+The snippet below shows a part of a document with the annotated results:
+
+[source,console-result]
+----  
+          ...
+          "FlightDelay" : false, <1>
+          "DistanceMiles" : 8865.200239823978,
+          "FlightTimeMin" : 792.6198230421824,
+          "OriginWeather" : "Cloudy",
+          "dayOfWeek" : 1,
+          "AvgTicketPrice" : 816.1058877497752,
+          "Carrier" : "Kibana Airlines",
+          "FlightDelayMin" : 0,
+          "OriginRegion" : "CO-CUN",
+          "FlightDelayType" : "No Delay",
+          "DestAirportID" : "NRT",
+          "timestamp" : "2019-11-12T17:06:35",
+          "Dest" : "Narita International Airport",
+          "FlightTimeHour" : 13.210330384036373,
+          "Cancelled" : false,
+          "DistanceKilometers" : 14267.156814759282,
+          "OriginCityName" : "Bogota",
+          "DestWeather" : "Sunny",
+          "OriginCountry" : "CO",
+          "ml__id_copy" : "-7OQg24B95sXiIrm0EZo",
+          "DestCountry" : "JP",
+          "DestRegion" : "SE-BD",
+          "OriginAirportID" : "BOG",
+          "DestCityName" : "Tokyo",
+          "ml" : {
+            "top_classes" : [ <2>
+              {
+                "class_probability" : 0.939335365058496, <3>
+                "class_name" : "false" <4>
+              },
+              {
+                "class_probability" : 0.06066463494150393,
+                "class_name" : "true"
+              }
+            ],
+            "FlightDelay_prediction" : "false", <5>
+            "is_training" : false <6>
+          }
+----
+<1> The `dependent_variable` with the ground truth value. This is what we are 
+trying to predict with the {classanalysis}.
+<2> The top classes object contains the predicted classes with the highest 
+probability.
+<3> The probability that the datapoint belongs to the given class. This is a 
+value between 0 and 1. The higher the number the higher the probability that the 
+datapoint belongs to the class.
+<4> Name of the class.
+<5> The prediction. The field name is suffixed with `_prediction`.
+<6> Indicates that this document was not used in the training set.
+
+The example above shows that the analysis has predicted the probability of all 
+the possible classes. In this case, there are two classes: `true` and `false`. 
+The class names along with the probability of the given classes are displayed in 
+the `top_classes` object. The most probable class is the prediction. In the 
+example above, `false` has a `class_probability` of 0.93 while `true` has only 
+0.06, so the prediction will be `false` which coincides with the ground truth 
+contained by the `FlightDelay` field. The class probability values help you to 
+understand how sure the model is about the prediction. The higher number means 
+that the model is more confident.
+
+If a document doesn't contain a prediction field, then it is excluded from the 
```

Comment:
"If a document doesn’t contain a prediction field, then it is excluded from the analysis."

Please clarify what do you mean by a "prediction field"? If you mean "dependent variable" then the sentence is not true.